crates/network-scanner/Cargo.toml

```diff
@@ -8,12 +8,20 @@ publish = false
 
 [dependencies]
 anyhow = "1.0.79"
+crossbeam = { version = "0.8.4", features = ["crossbeam-channel"] }
+dashmap = "5.5.3"
+derive_builder = "0.13.0"
+futures = { version = "0.3.30" }
+lazy_static = "1.4.0"
```

Comment:
question: Unused?

Reply:
By the way, the standard library now has OnceLock which can be always be used instead of lazy_static: https://doc.rust-lang.org/std/sync/struct.OnceLock.html

Reply:
was trying to use it to have singleton runtime/task_manager, removed

Reply:
https://users.rust-lang.org/t/if-let-else-if-let-else-better-way-to-structure-code/104393/10?u=cbenoit